.travis.yml

```diff
@@ -9,8 +9,9 @@ dist: xenial
 language: rust
 
 install:
-  - nvm install 10 && nvm alias default 10 && npm install -g yarn@latest && yarn --version
-  - (cd dex-contracts && yarn --frozen-lockfile && yarn run prepack)
+  - nvm install 12 && nvm alias default 12 && npm install -g yarn@latest && yarn --version
+  # Yarn exits with non zero exit code because an optional dependency does not compile.
+  - (cd dex-contracts && yarn --frozen-lockfile && yarn run prepack || true)
```

Comment:
This shouldn't be necessary... I just tested locally and:
```
$ yarn
...
gyp ERR! cwd /var/home/nlordell/Developer/dex-contracts/node_modules/scrypt
gyp ERR! node -v v12.16.2
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok"
Done in 32.74s.
$ echo $?
0
```

Reply:
This doesn't seem to be the case on CI. I just tested it again and it definitely fails there without `||
true`.

Reply:
Hmm...

Reply:
Ah I figured out the issue! This is because the submodule version does not include the following commit, which is required for NodeJS 12 support:
https://github.com/gnosis/dex-contracts/commit/052ba1bc4f9c5467970ba671aca5f5e013cd6f9d

Reply:
Maybe we should first update the submodule to `v0.3.0` and then merge this PR. Otherwise I imagine it will not work with NodeJS 12.